app/models/concerns/offer_search.rb

```diff
@@ -25,7 +26,10 @@ module OfferSearch
           joins("LEFT OUTER JOIN items ON offers.id = items.offer_id").
           joins("LEFT OUTER JOIN messages ON offers.id = messages.offer_id OR items.id = messages.item_id").
           joins("LEFT OUTER JOIN packages ON packages.item_id = items.id").
-          joins("LEFT OUTER JOIN package_types ON package_types.id = packages.package_type_id")
+          joins("LEFT OUTER JOIN package_types ON package_types.id = packages.package_type_id").
```

Comment:
Place the `.` on the next line, together with the method name.

From code climate.

Reply:
don't do it @steveyken  code climate is evil 😆